src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String data = readDataFromFile(fromFileName);
+        writeDataToFile(toFileName, data);
+    }
 
+    private String readDataFromFile(String fromFileName) {
+        int countSupply = 0;
+        int countBuy = 0;
+        int countResult = 0;
+        StringBuilder report = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                String[] splitData = line.split(",");
+                switch (splitData[0]) {
```

Comment:
![Screenshot 2022-06-21 at 08 12 19](https://user-images.githubusercontent.com/60258373/174720976-ceb22b73-9b74-422f-bb6c-b22cb721648c.png)
Before you create PR with your homework, you need to go through the checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist items are corrected